common/src/sim_config.rs

```diff
@@ -37,6 +39,7 @@ pub struct SimConfig {
     pub view_distance: f32,
     pub input_queue_size: Duration,
     pub chunk_size: u8,
+    pub gameplay_enabled: bool,
```

Comment:
This is currently documented in `SimConfigRaw`. I left this undocumented for consistency? Should I go ahead and document all of these duplicated fields. It would be easier when navigating the rest of the code in an IDE, but there's always the risk of these two duplicate pieces of documentation going out of sync.